starknetdata/feeder/feeder.go

```diff
@@ -80,6 +80,7 @@ func adaptBlock(response *feeder.Block) (*core.Block, error) {
 			SequencerAddress: response.SequencerAddress,
 			TransactionCount: uint64(len(response.Transactions)),
 			EventCount:       eventCount,
+			EventsBloom:      core.EventsBloom(receipts),
```

Comment:
Should we modify the tests to account for this?

Reply:
I think only thing we can test is that the EventsBloom is not nil. Behaviour of the bloom filter is tested in EventFilter already.